src/styles/card.scss

```
@@ -0,0 +1,59 @@
+.card {
+  display: flex;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
```

```suggestion
font-family: "Roboto", sans-serif;
```
You can add a font family for the entire card or even for the entire body to get it inherited by every element inside